preventing tiles flashing in wrong colors while data is loading for GithubCalendarWidget

JameDevOfficial opened the pull request:
## Description

Previously, the github calender widget flashed tiles with incorrect colors while the data wasn't loaded yet. Fixed this by passing the default theme.

## Type of Change

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update
- [ ] Translation update <!-- If your change is a Translation update, PLEASE run `npm run translation` before you start editing and after before commiting. Then check the box at the bottom of the checklist.-->

## Changelog Entry

- feat: add force dark mode for github calendar to prevent white frames. [0e807d4](https://github.com/JameDevOfficial/TablissNG/commit/0e807d4adf3fd8f7d67c8f5c590f4e6cc821798a)

## Checklist

- [x] Tested in development environment
- [x] Tested in Firefox
- [x] Tested in Chrome/Edge
- [x] Added or updated unit tests (if applicable)
- [x] All new and existing tests passed
- [x] I have added a changelog entry
- [x] I ran `npm run translations`
- [x] This PR is ready to be merged

JameDevOfficial commented:
It has a different gray tone while the data is not loaded in only dark mode

BookCatKid commented:
> It has a different gray tone while the data is not loaded in only dark mode

Where did you get the default colors from? Can I have a similar overide to fix the light mode loading? Do you have any idea on how to make the calendar not have to flash the non loaded content on every new tab? I wish it could cache itself or something.

JameDevOfficial commented:
## Default Colors:
from [this repo](https://github.com/grubersjoe/react-github-calendar):
```tsx
const gitHubTheme = {
  light: ['#ebedf0', '#9be9a8', '#40c463', '#30a14e', '#216e39'],
  dark: ['#161b22', '#0e4429', '#006d32', '#26a641', '#39d353'],
} satisfies ThemeInput
```

## Light Mode:
It doesn't seem like white has this issue because the tiles are white while unloaded (or at least not as noticeable), but you could use the colors from above

## Cache:
- I tried implementing it but couldn't get it to work, I have changed it now so it doesn't flash. You will now only see the white / black tiles (previously you could see a completely invisible widget and/or wrong colors while unloaded)

also reverted the translations commit since it is not used anymore and added a semicolon that vsc screamed at me was missing

BookCatKid commented on src/plugins/widgets/github/GitHub.tsx:
```diff
@@ -24,7 +26,7 @@ const GitHubCalendarWidget: FC<Props> = ({ data = defaultData }) => {
       weekdays.thu, weekdays.fri, weekdays.sat
     ],
-    totalCount: intl.formatMessage(messages.totalCount),
+    totalCount: (intl.messages[messages.totalCount.id] as string) || messages.totalCount.defaultMessage || "{{count}} contributions in {{year}}",
```
Can you explain why you made this change?

BookCatKid commented on src/plugins/widgets/github/GitHubSettings.tsx:
```diff
@@ -70,6 +70,7 @@ const GitHubSettings: React.FC<Props> = ({ data = defaultData, setData }) => (
         description="Label for click action dropdown"
       />
       <select
+        title="Click Action"
```
If you want to add this, please add translations.

JameDevOfficial replied:
Would you mind doing this on your end? It seems like I am replacing the line endings or something invisible that causes random changes that are changing nothing

BookCatKid commented:
Thank you for your pr, merging now!
